lib/DDG/Spice/SalesTax.pm

```diff
@@ -4,41 +4,49 @@ package DDG::Spice::SalesTax;
 use strict;
 use DDG::Spice;
 use Locale::SubCountry;
-use YAML::XS 'LoadFile';
 
-triggers any => 'sales tax for', 'sales tax', 'sales tax in';
-spice from => '(.*)/(.*)';
-spice to => 'https://taxrates.api.avalara.com/postal?country=usa&postal=$1&apikey={{ENV{DDG_SPICE_SALESTAX_APIKEY}}}';
+triggers any => 'sales tax for', 'sales tax in', 'sales tax';
+spice to => 'http://dev.snapcx.io:9100/taxv1/getStateTaxRate?request_id=DDG_Requestor&state=$1';
 spice wrap_jsonp_callback => 1;
+spice is_cached => 1;
 
 #Create US SubCountry object
 my $US = new Locale::SubCountry("US");
 
-#State to Zip Code
-my $zipcodes = LoadFile(share('zipcodes.yml'));
-
 # Handle statement
 handle remainder_lc => sub {
-    my ($state,$zip); #Define vars
+
+    return unless $loc && $loc->{country_code} eq 'US';
```

Comment:
This condition should only be required for the generic queries that show sales taxes for all states. Right now this IA has been limited only to US users, meaning anyone else in the world searching "sales tax in Texas" won't see the IA.